schemas/data/metric.schema.json

```diff
@@ -44,5 +44,5 @@
     }
   ],
   "required": ["@id", "schema:name", "xdm:measurement", "xdm:unit"],
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
👍

Reply:
👎
- It uses schema:name instead of xdm:name
- measurement needs to be an enum
- unit needs to be an enum